tests/dispatchers/sync.test.ts

```diff
@@ -25,7 +25,7 @@ describe('Dispatchers - Sync', () => {
       .mockImplementationOnce(() => Promise.resolve(JSON.stringify([dispatcher])));
 
     await expect(handler({ sdk: undefined, file: '' }))
-      .rejects.toThrow('Dispatcher name is a required field');
+      .rejects.toThrow('The dispatchers file is invalid');
```

Comment:
I feel like I'm being that guy today now... can we do the log spy and confirm what is actually wrong with validation?

It's not that robust of a test anymore, it could be any validation that has failed and not what the test specifies in both scenarios.